qiita_ware/test/test_ebi.py

```diff
@@ -530,7 +578,7 @@ def test_generate_demultiplexed_fastq(self):
       <STUDY_TITLE>
         Identification of the Microbiomes for Cannabis Soils
       </STUDY_TITLE>
-      <STUDY_TYPE existing_study_type="Metagenomics"/>
+      <STUDY_TYPE existing_study_type="Metagenomics" />
```

Comment:
Ahhh, I see, got it, thanks for the explanation!

On (Sep-04-15|12:10), Antonio Gonzalez wrote:
> > @@ -530,7 +578,7 @@ def test_generate_demultiplexed_fastq(self):
> >        <STUDY_TITLE>
> >          Identification of the Microbiomes for Cannabis Soils
> >        </STUDY_TITLE>
> > -      <STUDY_TYPE existing_study_type="Metagenomics"/>
> > +      <STUDY_TYPE existing_study_type="Metagenomics" />
> 
> Cause before the test was testing the XML (the test was converting test to XML and my guess is that the space was added there) and now we are testing the string that the XML generates. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1464/files#r38784801